imagetest/test_suites/README.md

```diff
@@ -0,0 +1,293 @@
+# Image test suites #
+
+## What is being tested ##
+
+The tests are a combination of various types - end to end tests on certain
+software components, image validations and feature validations, etc. The
+collective whole represents the quality assurance bar for releasing a [supported
```

Comment:
ditto

Reply:
ditty to the 'we need to clarify this' comment? you've selected the same three lines here